docs/getting_started.rst

```diff
@@ -0,0 +1,139 @@
+
+Projects using Zarr
+-------------------
+
+If you are using Zarr, we would `love to hear about it
+<https://github.com/zarr-developers/community/issues/19>`_.
+
+Acknowledgments
+---------------
+
+The following people have contributed to the development of Zarr by contributing code,
+documentation, code reviews, comments and/or ideas:
+
+* :user:`Alistair Miles <alimanfoo>`
+* :user:`Altay Sansal <tasansal>`
+* :user:`Anderson Banihirwe <andersy005>`
+* :user:`Andrew Fulton <andrewfulton9>`
+* :user:`Andrew Thomas <amcnicho>`
+* :user:`Anthony Scopatz <scopatz>`
+* :user:`Attila Bergou <abergou>`
+* :user:`BGCMHou <BGCMHou>`
+* :user:`Ben Jeffery <benjeffery>`
+* :user:`Ben Williams <benjaminhwilliams>`
+* :user:`Boaz Mohar <boazmohar>`
+* :user:`Charles Noyes <CSNoyes>`
+* :user:`Chris Barnes <clbarnes>`
+* :user:`David Baddeley <David-Baddeley>`
+* :user:`Davis Bennett <d-v-b>`
+* :user:`Dimitri Papadopoulos Orfanos <DimitriPapadopoulos>`
+* :user:`Eduardo Gonzalez <eddienko>`
+* :user:`Elliott Sales de Andrade <QuLogic>`
+* :user:`Eric Prestat <ericpre>`
+* :user:`Eric Younkin <ericgyounkin>`
+* :user:`Francesc Alted <FrancescAlted>`
+* :user:`Greggory Lee <grlee77>`
+* :user:`Gregory R. Lee <grlee77>`
+* :user:`Ian Hunt-Isaak <ianhi>`
+* :user:`James Bourbeau <jrbourbeau>`
+* :user:`Jan Funke <funkey>`
+* :user:`Jerome Kelleher <jeromekelleher>`
+* :user:`Joe Hamman <jhamman>`
+* :user:`Joe Jevnik <llllllllll>`
+* :user:`John Kirkham <jakirkham>`
+* :user:`Josh Moore <joshmoore>`
+* :user:`Juan Nunez-Iglesias <jni>`
+* :user:`Justin Swaney <jmswaney>`
+* :user:`Mads R. B. Kristensen <madsbk>`
+* :user:`Mamy Ratsimbazafy <mratsim>`
+* :user:`Martin Durant <martindurant>`
+* :user:`Matthew Rocklin <mrocklin>`
+* :user:`Matthias Bussonnier <Carreau>`
+* :user:`Mattia Almansi <malmans2>`
+* :user:`Noah D Brenowitz <nbren12>`
+* :user:`Oren Watson <orenwatson>`
+* :user:`Pavithra Eswaramoorthy <pavithraes>`
+* :user:`Poruri Sai Rahul <rahulporuri>`
+* :user:`Prakhar Goel <newt0311>`
+* :user:`Raphael Dussin <raphaeldussin>`
+* :user:`Ray Bell <raybellwaves>`
+* :user:`Richard Scott <RichardScottOZ>`
+* :user:`Richard Shaw <jrs65>`
+* :user:`Ryan Abernathey <rabernat>`
+* :user:`Ryan Williams <ryan-williams>`
+* :user:`Saransh Chopra <Saransh-cpp>`
+* :user:`Sebastian Grill <yetyetanotherusername>`
+* :user:`Shikhar Goenka <shikharsg>`
+* :user:`Shivank Chaudhary <Alt-Shivam>`
+* :user:`Stephan Hoyer <shoyer>`
+* :user:`Stephan Saalfeld <axtimwalde>`
+* :user:`Tarik Onalan <onalant>`
+* :user:`Tim Crone <tjcrone>`
+* :user:`Tobias Kölling <d70-t>`
+* :user:`Tom Augspurger <TomAugspurger>`
+* :user:`Tom White <tomwhite>`
+* :user:`Tommy Tran <potter420>`
+* :user:`Trevor Manz <manzt>`
+* :user:`Vincent Schut <vincentschut>`
+* :user:`Vyas Ramasubramani <vyasr>`
+* :user:`Zain Patel <mzjp2>`
+* :user:`gsakkis`
+* :user:`hailiangzhang <hailiangzhang>`
+* :user:`pmav99 <pmav99>`
+* :user:`sbalmer <sbalmer>`
+
+Contents
```

Comment:
Not sure "Contents" and below belongs under "Getting started"

Reply:
I think we should remove `Tutorials`, `API Reference`, `Specifications` and `Release Notes` from 'Contents' as they're already covered at the top.

We can leave `Contributing to Zarr` on 'Getting Started' if we want.

Also, maybe this is for me only, but I'm getting `404` for all of the various Tutorials and Release Notes when accessing from [here](https://zarr--1242.org.readthedocs.build/en/1242/getting_started.html).

Can you please check @DON-BRAN and @joshmoore?

Reply:
Yes, there's not really any need for the reiteration under `Getting started` though it could still be left so, but avoiding it looking bulky and getting it more organized, i suggest they could be taken out

@MSanKeys963 Everything opens fine for me, and looking really great

Reply:
> @MSanKeys963 Everything opens fine for me, and looking really great

👍🏻 

Can you please remove the sections mentioned above and add a separate page for acknowledgements?